sp_updater/config.py

```diff
@@ -4,7 +4,7 @@
 
-CONTEXT_CONTRACT_ABI = '[{"anonymous": false,"inputs": [{"indexed": false,"internalType": "address","name": "addr","type": "address"}],"name": "Sponsor","type": "event"}]'
+APP_CONTRACT_ABI = '[{"anonymous": false,"inputs": [{"indexed": false,"internalType": "address","name": "addr","type": "address"}],"name": "Sponsor","type": "event"}]'
```

Comment:
Why is this called `APP_CONTRACT_ABI`? Isn't this for setting the contract that emits `Sponsor` events?

Reply:
Applied :pray: